### PR TITLE
feat(statistic): statistic component 

### PR DESCRIPTION
**Pull request by James-9696**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/theme/src/statistic/smb-theme.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const tinyStatisticSmbTheme = {
+  'ti-statistic-font-size': '24px',
+  'ti-statistic-prefix-font-size': '24px',
+  'ti-statistic-font-color': '#191919',
+  'ti-statistic-suffix-font-size': 'var(--ti-common-font-size-4)'
+  // 'ti-statistic-suffix-font-weight': 'var(--ti-common-font-weight-3)'
```
Comment (zzcr) [inline packages/theme/src/statistic/smb-theme.js]: 无用的代码可删除

Comment (James-9696) [reply packages/theme/src/statistic/smb-theme.js]: 已删除

File: packages/vue/src/statistic/package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "@opentiny-internal/vue-test-utils": "workspace:*",
+    "vitest": "^0.31.0"
+  },
+  "scripts": {
+    "build": "pnpm -w build:ui $npm_package_name",
+    "//postversion": "pnpm build"
+  },
+  "dependencies": {
+    "@opentiny/vue-renderless": "workspace:~",
+    "@opentiny/vue-common": "workspace:~"
```
Comment (zzcr) [inline packages/vue/src/statistic/package.json]: dependencies需要补充完整，用到了哪些包都需要罗列出来，防止在pnpm工程依赖安装错误，这里应该还少theme相关依赖

File: packages/vue/src/statistic/index.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+ * Copyright (c) 2022 - present Huawei Cloud Computing Technologies Co., Ltd.
+ *
+ * Use of this source code is governed by an MIT-style license.
+ *
+ * THE OPEN SOURCE SOFTWARE IN THIS PRODUCT IS DISTRIBUTED IN THE HOPE THAT IT WILL BE USEFUL,
+ * BUT WITHOUT ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF MERCHANTABILITY OR FITNESS FOR
+ * A PARTICULAR PURPOSE. SEE THE APPLICABLE LICENSES FOR MORE DETAILS.
+ *
+ */
+import Statistic from './src/index.js'
```
Comment (zzcr) [inline packages/vue/src/statistic/index.ts]: 这里不能写index.js。都是ts的文件，直接写index就好了

File: packages/vue/src/statistic/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import type { PropType } from '@opentiny/vue-common'
+import type { StyleValue } from 'vue'
```
Comment (zzcr) [inline packages/vue/src/statistic/src/index.ts]: 这里在vue2会有问题，这种类型看看能不能从vue-common中透传一下？

**Review by Huangyilin19 — APPROVED** [review]

**Pull request merged**